ext/ddtrace.stub.php

```diff
@@ -82,6 +82,18 @@ class SpanData {
          */
         public string|null $service = "";
 
+        /**
+         * @var string The environment you are tracing. Defaults to active service at the time of span creation
+         * (i.e., the parent span), or datadog.env initialization settings if no parent exists
+         */
+        public string $env = "";
+
+        /**
+         * @var string The version of the application you are tracing. Defaults to active service at the time of
+         * span creation (i.e., the parent span), or datadog.version initialization settings if no parent exists
+         */
+        public string $version = "";
```

Comment:
These default to the "active service"? Is that right?

Reply:
Copy paste is a powerful tool. Powerful tools should be handled with more care than I apparently did :-D